sbnanaobj/StandardRecord/SRHelper.h

```diff
@@ -0,0 +1,11 @@
+#ifndef SBNANAOBJ_STANDARDRECORD_SRHELPER_H
+#define SBNANAOBJ_STANDARDRECORD_SRHELPER_H
+
+#include <limits>
+
+namespace caf
+{
+  const float kSignalingNaN = std::numeric_limits<float>::signaling_NaN();
```

Comment:
I _recommend_ setting it `constexpr`:
```suggestion
  constexpr float kSignalingNaN = std::numeric_limits<float>::signaling_NaN();
```
---- 
This other one is more for @cjbackhouse: is it ok to tie the name `kSignalingNaN` to a `float` value? do you foresee an use for the `double` counterpart?

Reply:
Updated! - Only tested in that it builds.

Reply:
Hmm. I do indeed think that we might one day want a double version (I normally resist doubles in the CAFs, but there are potentially legitimate reasons for one). If you assign this into a double, does the double end up NaN too, or not?
Since double will be so much rarer I think having a clunkier name like `kSignalingNaNDouble` for it would be ok.